pkg/mutation/assignmeta_mutator.go

```diff
@@ -82,6 +82,10 @@ func (m *AssignMetadataMutator) DeepCopy() Mutator {
 	return res
 }
 
+func (m *AssignMetadataMutator) Value() interface{} {
+	return string(m.assignMetadata.Spec.Parameters.Assign.Raw)
```

Comment:
The same logic must be done here too.
The content (in this case the string) is held on assign["value"]

Reply:
Done